docs/src/getting_started/inverse_lighting.md

```diff
@@ -0,0 +1,151 @@
+# Inverse Lighting Tutorial
+
+In this tutorial we shall explore the inverse lighting problem.
+Here, we shall try to reconstruct a target image by optimizing
+the parameters of the light source (using gradients).
+
+```julia
+using RayTracer, Images, Zygote, Flux, Statistics
+```
+
+## Configuring the Scene
+
+Reduce the screen_size if the optimization is taking a bit long
+
+```julia
+screen_size = (w = 300, h = 300)
+```
+
+Now we shall load the scene using [`load_obj`](@ref) function. For
+this we need the `obj` and `mtl` files. This will be downloaded using
```

Comment:
Again, a bit of description for both OBJ and MTL? 

"*They* will be downloaded"